.github/workflows/ruby-build.yml

```
@@ -254,7 +254,7 @@ jobs:
           # jq is available in epel-release (https://docs.fedoraproject.org/en-US/epel/)
           yum install -y gh unzip epel-release
           yum install -y jq
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

@criemen We should not bump the version for the centos build. Centos 7 is not compatible with node20.

```suggestion
      - uses: actions/checkout@v3
```